packages/scrollable-image/makefile

```diff
@@ -1,13 +1,31 @@
 ROOT_DIR ?= $(abspath ../..)
 include $(ROOT_DIR)/dev/base.makefile
 
-build: 
-	@echo "Temporay overwrite base.makefile build target for passing git commit check"
+help:
+	@echo "$(P) make build - Transpile es6 and above to es5 files, and build webpack bundle"
+	@echo "$(P) make test-embedded-code - Local build mock embedded codes to test"
 
-build-static-assets:
-	@echo "$(P) Build embedded code to dev/output.txt"
-	NODE_ENV=development $(ROOT_BIN_DIR)/babel-node --root-mode upward "./dev/local-build.js"
+build: clean build-default
+	@echo "$(P) Build distribution files to dist/"
```

Comment:
"Build webpack bundles, chunks and webpack-assets.json to dist" ?

Reply:
updated.